emcl2:
  ros__parameters:
    odom_freq: 20
    num_particles: 500
    odom_frame_id: "odom"
    footprint_frame_id: "base_footprint"
    base_frame_id: "base_link"
    odom_fw_dev_per_fw: 0.19
    odom_fw_dev_per_rot: 0.0001
    odom_rot_dev_per_fw: 0.13
    odom_rot_dev_per_rot: 0.2
    laser_likelihood_max_dist: 0.2
    alpha_threshold: 0.5
    open_space_threshold: 0.05
    expansion_radius_position: 0.1
    expansion_radius_orientation: 0.2
    range_threshold: 0.3
    laser_min_range: 0.0
    laser_max_range: 100000000.0
    scan_increment: 1
